src/PVRIptvData.cpp

```diff
@@ -825,15 +833,15 @@ PVR_ERROR PVRIptvData::GetEPGForChannel(ADDON_HANDLE handle, int iChannelUid, ti
       else
       {
         tag.iGenreType          = EPG_GENRE_USE_STRING;
-        tag.iGenreSubType       = 0;     /* not supported */
+        tag.iGenreSubType       = 0;   /* not supported */
```

Comment:
unrelated change. If you want to refactor do it in a separate commit

Reply:
reverted